lib/archethic/p2p/message.ex

```diff
@@ -429,8 +429,8 @@ defmodule Archethic.P2P.Message do
   def encode(%Balance{uco: uco_balance, nft: nft_balances}) do
     nft_balances_binary =
       nft_balances
-      |> Enum.reduce([], fn {nft_address, amount}, acc ->
-        [<<nft_address::binary, amount::float>> | acc]
+      |> Enum.reduce([], fn {{nft_address, nft_id}, amount}, acc ->
+        [<<nft_address::binary, amount::float, nft_id::8>> | acc]
```

Comment:
NFT Id is 8 bits so, it's range will be between 0-255? or is it 8 bytes?

Reply:
This is erlang bitstring notation.
So you can define how bits or binary are encoded.
Here it means the id is encoded in 8 bits

Reply:
So, In a NFT can have maximum of 256 properties,  aka NFT Tokens, if taken sequentially 0..255. If It's a hard restriction that is not mentioned in aeip #2 then we should mention it there as well.

Reply:
Yes good observation !
Actually the nft_id is limited to 256 values because we can have a maximum of 256 inputs in a transaction.
But we wanted to created a new issue to make the number of inputs dynamic ot have un unlimited number of input (or at least a huge number)

Reply:
![Screenshot from 2022-06-23 00-09-55](https://user-images.githubusercontent.com/89784724/175112214-be978f51-8d65-4996-a216-a2089e4d6c80.png)
Here Id: 2022 can't fit in 8bits so, we can change this in our aeip too. 
And I think there are some project who have 6000+ NFT tokens for a NFT, so having a bigger range would be definitely better, and yes with an encoding scheme mentioning a length and after that, all properties would make it huge. 
Got your point also with the maximum of 256 Input transactions, thanks for explaining.